dot/peerset/handler.go

```diff
@@ -77,6 +77,10 @@ func (h *Handler) RemovePeer(setID int, peers ...peer.ID) {
 
 // ReportPeer reports ReputationChange according to the peer behaviour.
 func (h *Handler) ReportPeer(rep ReputationChange, peers ...peer.ID) {
+	for _, pid := range peers {
```

Comment:
is there a way to check logging level? so we don't range across peers if package logging level isn't trace or debug.

Reply:
We always ever have a single peer for this method ([slightly related comment](https://github.com/ChainSafe/gossamer/pull/2435#discussion_r835155766)), so it's fine really.